.github/workflows/mac.yml

```
@@ -1,14 +1,14 @@
-name: macOS (macOS 12, Python 3.11)
+name: macOS (macOS 13, Python 3.11, x86_64 and ARM64)
```

```suggestion
name: macOS (Python 3.11)
```
because x86_64 or arm64 will be added by matrix